From c870489b518e64e585e0c8ce40e030bd44d7be91 Mon Sep 17 00:00:00 2001
From: "A. Unique TensorFlower" <gardener@tensorflow.org>
Date: Wed, 12 Aug 2020 14:51:05 -0700
Subject: [PATCH] grpc_impl namespace is gRPC internal and shouldn't be used by
 any application. Recommendation is to use grpc namespace. This CL makes the
 change.

PiperOrigin-RevId: 326317917
Change-Id: Ia915db210cdb84f9218561190893ed75f074186c
---
 tensorflow/core/data/service/data_service.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tensorflow/core/data/service/data_service.cc b/tensorflow/core/data/service/data_service.cc
index be09b10c1fc42..31449f6f5ec0a 100644
--- a/tensorflow/core/data/service/data_service.cc
+++ b/tensorflow/core/data/service/data_service.cc
@@ -117,7 +117,7 @@ Status DataServiceDispatcherClient::GetTasks(int64 job_id,
   GetTasksRequest req;
   req.set_job_id(job_id);
   GetTasksResponse resp;
-  grpc_impl::ClientContext ctx;
+  grpc::ClientContext ctx;
   grpc::Status s = stub_->GetTasks(&ctx, req, &resp);
   if (!s.ok()) {
     return grpc_util::WrapError("Failed to get tasks", s);
@@ -135,7 +135,7 @@ Status DataServiceDispatcherClient::GetWorkers(
   TF_RETURN_IF_ERROR(EnsureInitialized());
   GetWorkersRequest req;
   GetWorkersResponse resp;
-  grpc_impl::ClientContext ctx;
+  grpc::ClientContext ctx;
   grpc::Status s = stub_->GetWorkers(&ctx, req, &resp);
   if (!s.ok()) {
     return grpc_util::WrapError("Failed to get workers", s);
@@ -163,7 +163,7 @@ Status DataServiceWorkerClient::GetElement(int64 task_id,
   GetElementRequest req;
   req.set_task_id(task_id);
   GetElementResponse resp;
-  grpc_impl::ClientContext ctx;
+  grpc::ClientContext ctx;
   grpc::Status s = stub_->GetElement(&ctx, req, &resp);
   if (!s.ok()) {
     return grpc_util::WrapError("Failed to get element", s);
